Dev/Homebrew/package_manager.7h.py

```diff
@@ -214,7 +214,12 @@ def name(self):
     def sync(self):
         output = self.run(self.cli, '-g', '--progress=false', '--json',
                           'outdated')
+        if not output:
+            return
+
         for package, values in json.loads(output).iteritems():
+            if values['current'] == 'link':
```

Comment:
needs to be `values['wanted'] == 'linked'`. I know this seems very counter intuitive, but the `current` is the version that is in the package.json, which is likely never `linked`.